database.go

```diff
@@ -994,6 +1005,35 @@ func (db *database) MeasurementAndSeries(name string, tags map[string]string) (*
 	return idx, idx.seriesByTags(tags)
 }
 
+// SereiesByID returns the Series that has the given id.
+func (d *database) SeriesByID(id uint32) *Series {
+	return d.series[id]
+}
+
+// Names returns all measurement names in sorted order.
+func (d *database) Names() []string {
```

Comment:
I think this should be called `MeasurementNames`.  However, I wrote `func (db *database) MeasurementNames()`already and stupidly iterated through `db.measurements` keys.  Delete this func and fix mine?